docs/tutorial/auto-scaling.md

```diff
@@ -0,0 +1,127 @@
+Fleet allows you to easily manage your auto-scaling group whereby you can set a min/max of servers for a paritcular environment.
+
+When you describe an environment you can see min/max server pool limits in place as well as how many frontends a particular release is currently running:
+```bash
+$ fleet env describe prod
+----------------  -------------------------
+name              prod
+status            RUNNING
+whitelist         allow-all
+ssl certificate   prod
+created           2015-03-24 23:06:21+00:00
+updated           2015-06-30 05:53:09+00:00
+recycling         ON
+tracked branches  fleet-deploy
+previous release
+'autoscaling min   2'
+'autoscaling max   4'
+----------------  -------------------------
+
+Releases:
+name     status      loaded                     updated                      'frontends'
+-------  ----------  -------------------------  -------------------------  -----------
+037205f  * ACTIVE *  2015-06-30 05:37:51+00:00  2015-06-30 05:44:22+00:00            '2'
+
+Endpoints:
+--------  ------------------------------
+admin     admin.prod.ancora.f.nchr.io
+adminssh  adminssh.prod.ancora.f.nchr.io
+www       www.prod.ancora.f.nchr.io
+--------  ------------------------------
+```
+
+## Limits
+
+In the event that you need to serve more traffic you can simply expand your server pool 'limits' from 2/4 (min/max) to say 4/10 servers like so:
+
+```bash
+$ fleet env autoscaling limits prod 4 10
+```
+
+There are resources limits in place on your Fleet account for the total number of instances you can run. At any point you see for yourself how many instances you are running and what the current max amount is set to by running:
```

Comment:
resources -> resource (no plural needed there)

"At any point you see for yourself how many instances you are running and what the current max amount is set to by running"
Missing word? "At any point you **can** see for yourself"
Also, better to use number rather than amount for instances.